### ISSUE TITLE
PM4-2 The Savage (Ouryu)

### ISSUE THREAD

**Issue by PoppaPip** [EdenServer/community#2082]
Hi! Promathia Mission 4-2 The Savage BCNM acted strangely when we attempted a few weeks ago 9/13/2019.

The first attempt he appeared to have wild regain. He Typhoon Winged twice in 20 seconds, dealing over 400 damage to everyone (this was while he was on the ground).

The second attempt he had 100% evasion, we swung for probably 2-3 minutes (I counted 38 or so swings) before we gave up and exited the BCNM.

The third attempt he had 100% evasion, but according to a source on Discord this had happened to them, until after he flew then landed for the first time. We kited until he flew then landed, and when we re-engaged his evasion was "normal" and we were able to win.

### Checklist

- [x] I have searched for existing issues for issues like this one. The issue has not been posted. (Duplicate reports slow down development.)
- [x] I have provided reproducable steps. (No "as the title says" posts please. Provide reproducable steps even if it seems like a waste of time.)
- [x] I have provided my client version in the details. (type /ver into your game window)

### Details

version 30181205_0

**Comment by Slamm21** [comment]
The same just happened to me. We went in and he was impossible to hit until he flew and I brought him back down with a mistmelt. Then it was fine.

**Comment by Raizuo** [comment]
The same thing happened after the 12/30 patch. The first fight, we all wiped due to an insane tp chain, then AoE breath that did 1200-1300 to everyone in front of Ouryu. In the second fight, we encountered the 100% evasion bug where only our RNG did dmg for the first minute or two. Mistmisted after it flew then melee finally was able to damage it. We got the win shortly afterward. No crazy tp chain or 1300 breath attacks.
